velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -73,6 +73,23 @@ class SubstraitVeloxPlanConverter {
       const ::substrait::Plan& substraitPlan,
       memory::MemoryPool* pool);
 
+  /// Multiple conditions are connected to a binary tree structure with
+  /// the relation key words, including AND, OR, and etc. Currently, only
+  /// AND is supported. This function is used to extract all the Substrait
+  /// conditions in the binary tree structure into a vector.
+  void flattenConditions(
```

Comment:
Does this method need to be public? Why?

Reply:
Moved back to private. Previously used by validation.